<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>561</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #F29F05;
}</string>
   </property>
   <property name="text">
    <string>Thanks for using my program!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>601</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	border-radius: 20px;
	background-color: #595645;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 20px;
	color: #595645;
	background-color: #F29F05;
}</string>
    </property>
    <property name="text">
     <string>OK!</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>561</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-color: #665C4A;
	border-radius: 20px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: #F2C744;
}</string>
    </property>
    <property name="text">
     <string>Brief instructions for use:
1. Paste the text into the text.txt
2. Wait for the download to finish and click on the 
 NEW WORD (A) or OLD WORD (A) buttons, depending on
 what it is for you.
 Keyboard shortcuts also work:
 D - new word
 A - old word
 Z - step back
 F1 - open instruction
3. In case of a wrong choice, click the Step Back (Z),
 so you can go back to the previous words.
4. At the end of the word distribution, in the file new.txt
 will be new words with a translation into Russian
5. The words that marked as old will be in the
 notnew file.txt</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>label_1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
